salesagility/SuiteCRM
# Fix #9745 - Fix email signatures being replaced

Remove code that replaces email signatures





## Description
Fixes Issue 1 - Email signatures are not added when replying to email if an email signature has been added by SuiteCRM already.
Fixes Issue 2 - Existing email signatures in email thread are replaced on load by SuiteCRM if the email signature has been updated in the user profile, this will cause issues if other applications use the HTML class "email-signature" for their signature as SuiteCRM will replace it with the user who is composing the email. Same issue if two users in SuiteCRM are in an email thread together, the person composing the email will have their email signature replace the other users signature in the email thread.





## Motivation and Context
If email signature is already added in the thread then a new signature will not be added and will be replaced by the users current signature.


## How To Test This
1. Set-up outbound email
2. Add an email signature in the user profile
3. Send email to inbound mailbox
4. Update email signature to be different from what was originally set
5. Check email from within SuiteCRM and "reply to"
6. See that email signature is added and not signatures are replaced


## Types of changes


### Final checklist
